docs/it/configurazione_federazione.rst

```diff
@@ -8,7 +8,7 @@ Nella Federazione OIDC il **Trust Anchor** è il servizio gestito dalla Autorit
 
 La configurazione della Federazione è pubblicata dal Trust Anchor all’interno della sua :ref:`Entity Configuration<entity_configuration_ta>`, disponibile presso un web path ben noto e corrispondente a **.well-known/openid-federation**.
 
-Tutti i partecipanti DEVONO ottenere prima della fase di esercizio la configurazione della Federazione e mantenere questa aggiornata su base giornaliera. All’interno della Configurazione della Federazione è presente la chiave pubblica del Trust Anchor usata per le operazioni di firma, il numero massimo di Intermediari consentiti tra una Foglia e il Trust Anchor (**max_path length**) e le autorità abilitate all’emissione dei Trust Marks (**trust_marks_issuers**).
+Tutti i partecipanti DEVONO ottenere prima della fase di esercizio la configurazione della Federazione e mantenere questa aggiornata su base giornaliera. All’interno della Configurazione della Federazione è presente la chiave pubblica del Trust Anchor usata per le operazioni di firma, il numero massimo di Intermediari consentiti tra una Foglia e il Trust Anchor (**max_path length**) e le autorità abilitate all’emissione dei Trust Mark (**trust_marks_issuers**).
```

Comment:
Trust Mark è sempre singolare, perché il testo è in italiano. Nella versione in inglese va invece al plurale